core/trie/proof.go

```diff
@@ -54,6 +54,11 @@ func (e *Edge) Len() uint8 {
 	return e.Path.Len()
 }
 
+func (e *Edge) PathInt() uint64 {
```

Comment:
Is this function still needed?

Reply:
It depends how this is decided https://github.com/starkware-libs/starknet-specs/pull/232#discussion_r1791659441

Reply:
This is still unused / unneeded?